features/facebookImport/src/popUps/infoPopUps/activitiesInfo.jsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+
+import i18n from "../../i18n.js";
+import BaseInfoPopUp from "../baseInfoPopUp/baseInfoPopUp.jsx";
+import Infographic from "../../components/infographic/infographic.jsx";
+
+import "./infoPopUps.css";
+
+const ActivitiesInfoScreen = () => {
+    const activitiesInfoText = [
+        <>
+            <p>{i18n.t("activitiesInfoScreen:text1")}</p>
```

Comment:
This is simply a `p`? Not a header?

Reply:
It's not a header, it's a paragraph. You can find the header in the `BaseInfoPopUp` component.

Reply:
Well, it's at the top, so... Anyway, fair enough.